thundra/wrappers/web_wrapper_utils.py

```diff
@@ -46,11 +47,13 @@ def start_trace(execution_context, tracer, class_name, domain_name, request):
     execution_context.scope = scope
     execution_context.trace_id = trace_id
 
-    trigger_operation_name = request.get('headers').get(constants.TRIGGER_RESOURCE_NAME_TAG) or \
+    trigger_operation_name = request_route_path or request.get('headers').get(constants.TRIGGER_RESOURCE_NAME_TAG) or \
```

Comment:
I think if `request_route_path` is exist, trigger operation name should be `request.get('host', '') + request_route_path` like we did for `normalized_path` (`request.get('host', '') + normalized_path`)